flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is disappointing.

Reply:
Not that I think about it, it's not necessary either. Electron IS available in other platforms...